workflow/tests/hpxml_translator_test.rb

```diff
@@ -176,6 +176,13 @@ def test_invalid
                                                             "The building is of type 'single-family detached' but the surface",
                                                             "The building is of type 'single-family detached' but the object",
                                                             "The building is of type 'single-family detached' but the HVAC distribution"],
+                            'invalid-input-parameters.xml' => ["Expected value to be: //XMLTransactionHeaderInformation/Transaction: 'create' or 'update'",
+                                                               "Expected value to be: //Building/BuildingDetails/BuildingSummary/Site/SiteType: 'rural' or 'suburban' or 'urban'",
+                                                               "Expected value to be: //Building/BuildingDetails/ClimateandRiskZones/ClimateZoneIECC/Year: '2012' or '2009' or '2006' or '2003'",
+                                                               'Expected value to be: //Building/BuildingDetails/Enclosure/Roofs/Roof/Azimuth: less than 360',
+                                                               'Expected value to be: //Building/BuildingDetails/Enclosure/Roofs/Roof/RadiantBarrierGrade: less than or equal to 3',
+                                                               'Expected value to be: //Building/BuildingDetails/Appliances/Dishwasher/EnergyFactor: less than or equal to 5',
+                                                               'Expected 1 element(s) for xpath: /HPXML/Building/BuildingDetails/BuildingSummary/Site: [not(SiteType)] | SiteType[text()="urban" or text()="suburban" or text()="rural"]'],
```

Comment:
Maybe now is the right time to reorganize our messages a bit. How about this? Or open to other ideas.
- "Expected value to be 'foo' or 'bar' [context: //Whatever/...]"
- "Expected value to be greater than or equal to 0 [context: //Whatever/...]"
- "Expected 1 element(s) for xpath: 'Foo | Bar' [context: //Whatever/...]"

Reply:
I like this idea. I can't think of any other way. I'll change messages in this PR and #483. Thanks!

Reply:
Thanks @bpark1327. I'll look this over one more time after you do, but I think it might be ready (or _very_ close) to merge.